src/main/webapp/i18n/de/featureOverview.json

```diff
@@ -70,6 +70,16 @@
                     "title": "Statistiken Erstellen",
                     "shortDescription": "Statistiken erstellen und Einblicke in die Leistungen der Studierenden gewinnen.",
                     "descriptionTextOne": "Die Klausur ist vorbei, alle Klausuren wurden bewertet und die Beschwerden der Studierenden wurden ausgewertet. Was nun?\n\nDu kannst das Ergebnis der Klausur anhand der von Artemis generierten Statistiken auswerten. Überprüfe die Leistung der Studierenden anhand verschiedener Metriken wie Durchschnitt, Median und Standardabweichung. Überprüfen Sie die wahrgenommenen Schwierigkeiten der Studierenden bei jeder Aufgabe, um die Klausuren in Zukunft zu verbessern. Zusätzlich kannst du alle Ergebnisse in eine CSV-Datei herunterladen, um sie in externe Systeme zu exportieren und Diagramme und Grafiken zur weiteren Analyse zu erstellen."
+                },
+                "checklist": {
+                    "title": "Checkliste Befolgen",
```

Comment:
```suggestion
                    "title": "Checkliste befolgen",
```

Reply:
It seemed to me that the other titles in the same file use capital case except for prepositions and articles etc. But my German is beginner-lever so if using lowercase would be better I can change this (and the one below).